students/macarena-sotomayor/ejericicios-dia20/script.js

```diff
@@ -0,0 +1,148 @@
+// // ejericicio 1,: crear dos variables prompr y que se represente la mayor de ellod
+
+// let a= prompt('escribir numero')
+// let b= prompt('escribir otro numero')
+
+
+
```

Comment:
Be careful `a` and `b` are strings and comparation beetween strings are different than numbers. Example '10' < '2' returns true